packages/sign_in_with_apple/sign_in_with_apple/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt

```diff
@@ -91,7 +91,6 @@ public class SignInWithApplePlugin: FlutterPlugin, MethodCallHandler, ActivityAw
 
         val builder = CustomTabsIntent.Builder();
         val customTabsIntent = builder.build();
-        customTabsIntent.intent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
```

Comment:
So the fix is to keep the activity on the history?

I can't exactly recall why we added this in the initial version, do you?

Reply:
Tested both deep links (SiwA and normal app (main activity) ones), and they would with both `launchMode`s close the CCT. For the in-app deep link the deep link handler would be called inside the app. 

So this seems fine as far as I can tell.